### DISCUSSION TITLE
High mouse refresh rate causes studders

### DISCUSSION THREAD

**Discussion by distrair** [ppy/osu#23543] [Q&A] [answered]
### Type

Performance

### Bug description

Game starts to studder when mouse refresh rate is set to 1000hz, when set to 125hz, no studders (with 500hz game have less studders but they still appear)

### Screenshots or videos

1000hz
https://www.youtube.com/watch?v=TRsEKCOvms4
125hz
https://www.youtube.com/watch?v=KxSPVEasGIc

### Version

2023.511.0

### Logs

[updater.log](https://github.com/ppy/osu/files/11470471/updater.log)
[performance.log](https://github.com/ppy/osu/files/11470472/performance.log)
[database.log](https://github.com/ppy/osu/files/11470473/database.log)
[input.log](https://github.com/ppy/osu/files/11470474/input.log)
[runtime.log](https://github.com/ppy/osu/files/11470475/runtime.log)
[network.log](https://github.com/ppy/osu/files/11470476/network.log)

**Comment by peppy** [comment, marked as answer]
99% going to be something else on your PC handling mouse incorrectly.

Either try and figure out what is causing it, or use a lower polling rate.
